tests/UI/campaigns/functional/API/01_basicTest.ts

```diff
@@ -23,12 +23,5 @@ describe('API : Basic Test', async () => {
       const apiResponse = await apiContext.get('');
       expect(apiResponse.status()).to.eq(404);
     });
-
-    it('should request the endpoint /hook-status', async function () {
```

Comment:
Please replace hook-status by hooks

(We don't test the endpoint but the access)